scripts/zones/Davoi/npcs/_45d.lua

```diff
@@ -25,7 +25,13 @@ end
 entity.onEventFinish = function(player, csid, option, npc)
     if csid == 42 and option == 0 then
         player:messageSpecial(ID.text.POWER_OF_THE_ORB_ALLOW_PASS)
-        npc:openDoor(16)
+        npc:openDoor(20)
+
+        -- Animation for the "door" being open
+        npc:setAnimation(8)
+        npc:timer(1000 * 20, function(npcArg)
```

Comment:
`openDoor()` itself is just doing animation changes, since its normal changes are incorrect here and you are setting the correct ones via `setAnimation()`, there is no more need for the command at all.

```cpp
        m_PBaseEntity->animation = ANIMATION_OPEN_DOOR;
```
There is also an update packet sent, but the same thing happens when we `setAnimation()`

Reply:
oh actually no, scratch that because:
https://github.com/LandSandBoat/server/blob/base/scripts/enum/animation.lua#L16

Reply:
was going to say use the enum, imagine my surprise